src/App.vue

```diff
@@ -146,6 +152,56 @@ export default {
     // https://vuex.vuejs.org/guide/state.html#object-spread-operator
     ...mapActions("auth", ["unsetAccessToken", "fetchAndUpdateUser"]),
     ...mapActions("sync", ["stopLoading"]),
+    mountChatwoot() {
+      // mounting chatwoot SDK to the DOM
+      let chatwootScript = document.createElement("script");
```

Comment:
being more specific (script is too broad)
```suggestion
      let chatwootScript = document.createElement("chatwoot-script");
```

Reply:
This is creating a `<script></script>` tag in HTML
There's nothing like a `<chatwoot-script></chatwoot-script>` tag in HTML